src/components/Gallery/Gallery.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+// import './Gallery.css';
+
+const galleryStyle = {
+  display: 'flex',
+  'flexDirection': 'row',
+  // 'flex-wrap': 'wrap'
+};
+
+export default class Gallery extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      images: [
+        {id: 1, url: "https://s3.amazonaws.com/uifaces/faces/twitter/kuldarkalvik/128.jpg" },
```

Comment:
I realize this is probably a placeholder, perhaps consider keeping example data in a separate file next time